### DISCUSSION TITLE
Milvus2.0 performance configuration

### DISCUSSION THREAD

**Discussion by WordlessToContinue** [milvus-io/milvus#16024] [Q&A and General discussion] [answered]
I have two machines with the same configuration  to test milvus0.10.1 and milvus2.0.1

1. test datas : 5 million 512dim float data
2. index params : HNSW    "M": 48,"efConstruction": 500
3. search params : "ef":300
4. operation ：create collection -> insert data -> create index -> search(nq = 100)

- milvus0.10.1
used one machine
index_file_size set 4096
search time: 156 ms

- milvus2.0.1
used two machines ,both machines are deployed querynode,datanode,indexnode
used the default configration
search consistency_level : "Eventually"
search time: 587 ms

Why is search time so different ？ Can I  make the milvus2.0.1 search time is close to half of milvus0.10.1 by modifying the configuration ？

**Comment by JackLCL** [comment, marked as answer]
In Milvus2.0.x, segment_size is default 512MB. And you can set the segment_size=4096MB for Milvus2.0 and test it again.
Here is the reference documentation: https://milvus.io/docs/v2.0.x/configure_datacoord.md#dataCoordsegmentmaxSize.
